cli/cmd/get.go

```diff
@@ -627,7 +627,7 @@ func strMapToStr(strings map[string]string) string {
 	return out
 }
 
-func dataResourceTable(resources []context.Resource, dataStatuses map[string]*resource.DataStatus) string {
+func dataResourceTable(resources []context.Resource, dataStatuses map[string]*resource.DataStatus, title string) string {
```

Comment:
Maybe `title` should be type `resource.Type`, and in `resource/type.go` we should have a function to get the user-facing type string (and a corresponding map similar to `types` and `typePlurals`?